.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1738,26 +1373,70 @@ public class FineTuningClient {
         public virtual ClientResult GetJobs(string after, int? limit, RequestOptions options);
         public virtual Task<ClientResult> GetJobsAsync(string after, int? limit, RequestOptions options);
     }
+    public class FineTuningJob {
+        public DateTimeOffset CreatedAt { get; }
+        public FineTuningJobError Error { get; }
+        public DateTimeOffset? EstimatedFinish { get; }
+        public string FineTunedModel { get; }
```

Comment:
Mostly just a note for later: we'll want to do a holistic naming pass to see what, if any, non-trivial naming deviations we'd like to make from the REST. I think "CreatedModelName" or similar here would be a lot more descriptive and accurate in a .NET API (similarly, something like "baseModelName" on the input) but that has to be weighed against spec/docs divergence and the likely level of existing knowledge/background fine-tuning customers will already have.